.codecov.yml

```diff
@@ -1,3 +1,4 @@
+---
```

Comment:
is it possible to disable the triple-dash rule? It seems to be quite common in many yaml-files (like in our case) to omit them (I also would say they can be count to redundant information, so that's something I would argue about in the YAML-standards themselves ;D)

Reply:
@shahidikram0701 What do you think 🤔

Reply:
Sorry for the late reply.
YAML spec says: YAML uses three dashes (---) to separate directives from document content. This also serves to signal the start of a document if no directives are present.

Reply:
But is it possible to disable them? It seems that all usages had no problems parsing YAML without those `---`^^

Reply:
According to what I read, When there are multiple documents in a YAML file it's necessary and when there is just a single document it can be omitted.